common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -438,6 +445,5 @@ nav.sequence-bottom {
 
 /*  &:focus {
     outline: 0;
-  }
-*/
+*/  }
```

Comment:
@clrux, looks like @cptvitamin did you a solid with https://github.com/edx/edx-platform/commit/e75fe342dfca9f56da0d98b10914dc047400e01d. Can you double-check that this change 1) solves the compilation problem and 2) preserves proper Sass nesting and indentation structure?